behavior/plans/diff.py

```diff
@@ -468,8 +468,12 @@ def main(data_dir, output_dir, experiment) -> None:
 
     for mode in ["train", "eval"]:
         experiment_root: Path = data_dir / mode / experiment
+        # The data folders must have prefix as the selected benchmark names.
+        # Its suffix is the concatenated parameters.
```

Comment:
Can you give one example data folder name here?
```suggestion
        # Its suffix is the concatenated parameters.
        # For example, `tpcc_scalefactor_0.1_terminals_1_rate_10000_time_60`.
```